homework3.py

```diff
@@ -0,0 +1,180 @@
+import random
+
+###############################################################
+# Problem 0: Write a program that takes an integer input from the user and checks if it's odd or even. Use an if-else
+# statement to print the result
+# ##############################################################
+
+number = int(input("Please enter a numer:"))
+
+if number % 2 == 0:
+    print("The number is even")
+else:
+    print("the number is odd")
+
+# ##############################################################
+# Problem 1:
+# Write a Python program to find the sum of all even numbers from 1 to 100 using a loop. Make
+# use of control flow constructs like the for loop and conditional statements.
+# ##############################################################
+sum = 0
+for i in range(0, 101):
+    if i % 2 == 0:
+        sum += i
+print("Sum is :", sum)
+
+# ##############################################################
+# Write a Python script that prompts the user in the console a simple problem ( how much does 5
+# + 17 equal to ) until the user provides a correct answer.
+
+result = int(input("How much does 5 + 17 equal to "))
+
+while True:
```

Comment:
В този случай съм съгласен, че може би този вариант е дори по-разбираем и прост. Въпреки това е интересно да се види алтернативата:

```suggestion
result = 0
correct_answer = 17

while result != correct_answer:
     print("Please enter new answer")
     result = int(input("Input a number : "))
     
print("The answer is correct! ")
```